server/src-lib/Hasura/GraphQL/Schema.hs

```diff
@@ -1717,10 +1717,10 @@ checkSchemaConflicts gCtx remoteCtx = do
 
     hQRName = G.NamedType "query_root"
     hMRName = G.NamedType "mutation_root"
-    tyMsg ty = "types: [" <> namesToTxt ty <>
-               "] already exist in current graphql schema"
-    nodesMsg n = "nodes : [" <> namesToTxt n <>
-                 "] already exist in current graphql schema"
+    tyMsg ty = "types: <" <> namesToTxt ty <>
+               "> have mismatch with current graphql schema. HINT: Types must be same."
+    nodesMsg n = "top-level nodes : <" <> namesToTxt n <>
```

Comment:
use the square brackets `[`, `]` itself, because this is a list of types?

Reply:
True. But its much clearer (IMHO) if its `< >` because with `[ ]` there is a confusion when the frontend throws error with just ONE type mismatch. 

It comes like `type [user] is conflicting`, which is confusing because I have the type user and also a return type of [user] .